apps/web/tailwind.config.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
         bcDisabled: '#CFCFCF',
         bcYellowCream: '#F9F1C6',
         bcYellowCreamStroke: '#FAEBCC',
+        bcDarkBlue: '#001a33',
```

Comment:
Can we make sure the color matches the standard here?

Reply:
Khan confirmed that this meets the BC guidelines, leaving it as is.